samples/features/high availability/Linux/Ansible Playbook/library/mssql_conf.py

```diff
@@ -116,7 +117,17 @@ def main():
 
 	if setup_sa_password is not None:
 		if os.path.isfile('/var/opt/mssql/mssql.conf'):
-			changed = False
+			msconfigfile = open('/var/opt/mssql/mssql.conf', 'r') 	
+			msconfigfiledata = msconfigfile.read()
+			msconfigfileresult = re.search(r'\b(accepteula)\b', msconfigfiledata, re.I)
+			msconfigfile.close()
+			if msconfigfileresult is None:
+				setup_env = os.environ.copy()
```

Comment:
Instead of duplicating this, merge it with the section below by extracting the "does mssql.conf exist with accepteula" into its own if-block.

```
already_setup = True
if os.path.isfile(...):
    if msconfigfileresult is None:
        already_setup = False

if not already_setup:
    suboprocess.check_call(...)
```